docs/formats.md

```diff
@@ -24,25 +23,61 @@ file under platforms > [platform] > files > [file]
 }
 ```
 
+### Using a template / templating engine to create a format
 
->*__How are Templates different than Formats?__*
+While a formatter is just a simple function, they are created easily with most templating engines. You might want to use a template if you have a lot of boilerplate code around where the style dictionary will go (e.g. ObjectiveC files). Writing a formatter function directly may be easier if there is little to no boilerplate code (e.g. a flat SCSS variables file).
 
->Mainly syntactic sugar; anything you can do in a Template you can do in a Format. Use whichever is easier for you to write. We find
-that Templates are good if you have a lot of boilerplate code around where the style dictionary will go (like writing ObjectiveC files).
-Formats are better if there is little to no boilerplate code like a flat SCSS variables file.
+Any templating language can work as there is a node module for it.
+All you need to do is register a custom format which calls your template and returns a string. Here is a quick example for Lodash.
+
+```js
+const StyleDictionary = require('style-dictionary').extend('config.json');
+const _ = require('lodash');
+
+_.template( fs.readFileSync( options.template ) )
+const template = _.template( fs.readFileSync('templates/MyTemplate.template') );
+
+styleDictionary.registerFormat({
+  name: 'my/format',
+  formatter: template
+});
+
```

Comment:
I think that here, or you show how `my/format` is used, or is better to remove the `styleDictionary.buildAllPlatforms();` line, it may be misleading

Reply:
yeah, I see what you mean. Do you think I should make up an example? Like sass/map? Or would that confuse people more?

Reply:
for me, since you are talking about _creating_ a format, you don't need to show also how to _use_ it, it's too much information. keep it simple :)

Reply:
Let me know if you like the new version.